### ISSUE TITLE
7zip 64 Error: Installing 2 packages / Installing NpackdCL 1: Hash sum (SHA1) ... was expected. The file has changed.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/windows-package-manager#194]
```
What steps will reproduce the problem?
1. 7-ZIP 64 bit installation
2.
3.

What is the expected output? What do you see instead?
Error: Installing 2 packages / Installing NpackdCL 1: Hash sum (SHA1) 
4b3343d448895df57ea4e1623f0eb695f92d9265 found, but 
ddee619df78bcda28462ac640a2a69c6443f3456 was expected. The file has changed.

What version of the product are you using? On what operating system?
Npackd64-1.17.9

Please provide any additional information below.
```

Original issue reported on code.google.com by `cmtopi...@gmail.com` on 29 Jan 2013 at 8:35

**Comment by GoogleCodeExporter** [comment]
```
Any package that depends on npackdcL seems to get the same error. Thanks

Operating system is Windows 7 64 bit.
```

Original comment by `cmtopi...@gmail.com` on 29 Jan 2013 at 9:02

**Comment by GoogleCodeExporter** [comment]
```
Another example is trying to install squirrel sql calls for:

4 package(s) will be installed: NpackdCL 1.16.4, NpackdCL 1, 7-ZIP A 4.65, 
Squirrel SQL Standard 3.4

Get same error:

Installing NpackdCL 1
Hash sum (SHA1) 4b3343d448895df57ea4e1623f0eb695f92d9265 found, but 
ddee619df78bcda28462ac640a2a69c6443f3456 was expected. The file has changed.
```

Original comment by `cmtopi...@gmail.com` on 30 Jan 2013 at 3:52

**Comment by GoogleCodeExporter** [comment]
```
If I uninstall npackdCL first I get the same error. After the install fails 
npackd CL 1.164 is installed, along with npackd package manager 1.17.9 and 
npackd [MSI database] Npackd GUID: {4c4e86de-f7bb-417b-b0fe-527c2888eef4} 1.17.9
```

Original comment by `cmtopi...@gmail.com` on 30 Jan 2013 at 3:56

**Comment by GoogleCodeExporter** [comment]
```
And npackd installer helper 1.1
```

Original comment by `cmtopi...@gmail.com` on 30 Jan 2013 at 3:56

**Comment by GoogleCodeExporter** [comment]
```
it works for me. Can you download the following file with your browser:
http://dl.dropbox.com/u/17046326/files/org.googlecode.windows-package-manager.Np
ackdCL-1.0.zip
?
```

Original comment by `tim.lebe...@gmail.com` on 30 Jan 2013 at 6:19

**Comment by GoogleCodeExporter** [comment]
```
That site is blocked by our proxy apparently. I don't have that entry in my
settings. I've got
https://windows-package-manager.googlecode.com/hg/repository/Rep.xml

https://windows-package-manager.googlecode.com/hg/repository/Rep64.xml.
```

Original comment by `cmtopi...@gmail.com` on 30 Jan 2013 at 7:05

**Comment by GoogleCodeExporter** [comment]
```
https://windows-package-manager.googlecode.com/hg/repository/Rep.xml
only contains references (addresses, URLs) to the binaries like
http://dl.dropbox.com/u/17046326/files/org.googlecode.windows-package-manager.Np
ackdCL-1.0.zip
Many programs are hosted on the web sites of their respective vendors.

I am not sure how I can help you here. You should probably talk to your proxy 
administrator.
```

Original comment by `tim.lebe...@gmail.com` on 31 Jan 2013 at 5:48

**Comment by GoogleCodeExporter** [comment]
```
I show that NpackdCL-1.0 is already installed. Why do quite a few packages
require it again from dl.dropbox.com?
```

Original comment by `cmtopi...@gmail.com` on 31 Jan 2013 at 5:51

**Comment by GoogleCodeExporter** [comment]
```
Issue 195 has been merged into this issue.
```

Original comment by `tim.lebe...@gmail.com` on 31 Jan 2013 at 5:53

**Comment by GoogleCodeExporter** [comment]
```
Sorry, npackd-1.16.4 is installed. Dependencies aren't maintained locally then?
```

Original comment by `cmtopi...@gmail.com` on 31 Jan 2013 at 6:03

**Comment by GoogleCodeExporter** [comment]
```
Dependencies are maintained locally, but some packages require NpackdCL in the 
version 1
```

Original comment by `tim.lebe...@gmail.com` on 31 Jan 2013 at 6:05

**Comment by GoogleCodeExporter** [comment]
```
Any chance you could send it to me another way, until I can get a response from 
my admins? Thanks again.
```

Original comment by `cmtopi...@gmail.com` on 31 Jan 2013 at 6:08

**Comment by GoogleCodeExporter** [comment]
```
it would not help. There is currently no way to install a manually downloaded 
package. All packages should be downloaded and installed via Npackd. You either 
unblock the sites where the software resides or host your own modified 
repository that points to "safe" sites. 

You can create your own repository by copying the default repository as it is 
also licensed under GPLv3.
```

Original comment by `tim.lebe...@gmail.com` on 31 Jan 2013 at 6:19

**Comment by GoogleCodeExporter** [comment]
Original comment by `tim.lebe...@gmail.com` on 10 Feb 2013 at 9:56
* Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
I see this has been closed, but I also experience this problem. I'm at a new 
company that blocks access to dropbox, Google Drive etc, so there are many 
items in the repository that I can't install. Can't you host it somewhere more 
'legit' (I mean that jokingly)?
```

Original comment by `igi...@gmail.com` on 14 May 2014 at 11:34

**Comment by GoogleCodeExporter** [comment]
```
Any suggestions? It could be possible to download all packages and make a copy 
of the repository where each URL shows to the new location.
```

Original comment by `tim.lebe...@gmail.com` on 14 May 2014 at 6:05

**Comment by GoogleCodeExporter** [comment]
```
Still related to my previous question. Let's look at MyUninstaller for example. 
Why did you store that package on Dropbox instead of just linking it directly 
to http://www.nirsoft.net/utils/myuninst.zip ?
```

Original comment by `igi...@gmail.com` on 20 Oct 2014 at 7:42

**Comment by GoogleCodeExporter** [comment]
```
Some packages are re-uploaded to Dropbox (for commercial packages) or to 
Sourceforge (for open source packages) if they do not provided version specific 
URLs or just stable URLs.

In case of MyUninstaller the download URL is not version specific.
```

Original comment by `tim.lebe...@gmail.com` on 20 Oct 2014 at 5:43
